### PR TITLE
Editing while reading

### PR DESCRIPTION
**Pull request by jean**
Fix some typos, formulations, and add some punctuation.

### CODE REVIEW

File: web/content/feature/teams.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: devradar Teams (Preview)
 ---
 
-Import your teams devradars and track progress in your team members competences.
+Import your team's devradars, and track progress in your team members' competencies.
```
Comment (anoff) [inline web/content/feature/teams.md]: Thanks for bringing `competences vs competencies` up. I never thought about it. After reading up a bit it seems like competences is the right word though. It seems to be the correct plural for competence:

competency --> competencies
competence --> competences

And looking at https://writingexplained.org/competence-vs-competency-difference it says that competency is just a legal-jargon version of competence.

Comment (jean) [reply web/content/feature/teams.md]: I must confess that I haven't encountered "competences" before. Looking at Google [ngrams](https://books.google.com/ngrams/graph?content=competences%2C+competencies&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Ccompetences%3B%2Cc0%3B.t1%3B%2Ccompetencies%3B%2Cc0), it looks like "competencies" is more common these days. 
The wikipedia article on [Competence (human resources)](https://en.wikipedia.org/wiki/Competence_(human_resources)) uses both interchangeably, "competencies" 64 times and "competences" 6 times. 
Wiktionary's page on [competence](https://en.wiktionary.org/wiki/competence) links to "competency" as a synonym.
 That said, "competences" certainly isn't wrong. If you prefer it, I'll revert.

Comment (anoff) [reply web/content/feature/teams.md]: Thanks for the ngram pointer. I checked also the [singular versions](https://books.google.com/ngrams/graph?content=competences%2Ccompetencies%2Ccompetence%2Ccompetency&case_insensitive=on&year_start=1980&year_end=2008&corpus=15&smoothing=3&share=&direct_url=t4%3B%2Ccompetences%3B%2Cc0%3B%2Cs0%3B%3Bcompetences%3B%2Cc0%3B%3BCompetences%3B%2Cc0%3B.t4%3B%2Ccompetencies%3B%2Cc0%3B%2Cs0%3B%3Bcompetencies%3B%2Cc0%3B%3BCompetencies%3B%2Cc0%3B%3BCOMPETENCIES%3B%2Cc0%3B.t4%3B%2Ccompetence%3B%2Cc0%3B%2Cs0%3B%3Bcompetence%3B%2Cc0%3B%3BCompetence%3B%2Cc0%3B.t4%3B%2Ccompetency%3B%2Cc0%3B%2Cs0%3B%3Bcompetency%3B%2Cc0%3B%3BCompetency%3B%2Cc0%3B%3BCOMPETENCY%3B%2Cc0) and there "competence" has way more hits than competency. If the pural theory above is correct it seems like it is used inconsistently. Wiktionary states "competences" as the plural of "competence". I would like to be consistent here with the chosen singular/plural alternative.

wrt to the plural I don't really have a preference; but for the singular one I really prefer "competence" - unless you have a compelling argument for competency. As a non-native english speaker it just seems a lot easier to pronounce competence.

Comment (jean) [reply web/content/feature/teams.md]: OK I made the plural consistent, and checked that "competency" does not occur.

**Comment by anoff** [comment]
Thanks again for your improvements 👍

**Pull request merged**